CONTRIBUTING.md

```diff
@@ -1,79 +1,114 @@
+
+## Create an issue
+
+If you notice a typo, slight bug in the code and there are no lodged complaints on this yet, select the issues icon at the top, and on the right side of the next page select new issues and lodge your complain.
 
-I have not created an official style guide yet. I also have not created a Figma or layout. If anyone would like to create a style guide for us. Create a Discussion. I have typically let the community help
+## Fork this repository
 
-## Development
+After choosing what to work on, move to fork and setup your own repository.
 
-If you are not sure what to work on, review the issues list. There are also TODO's that will listed in the included [TODO.md](./TODO.md) file.
+On the project's repository page, look to the right side of the page to locate a fork tab, click on this icon, and this would redirect you to a new page with further instructions. 
 
-React gives us built in hooks such as useEffect, useSelector, useRef, useHook, useCallback, useContext, useMemo, useForm, useHistory, etc. Please make sure that we make the most use of what React gives us.
+![](./docs/fork.png)
+
+On this next page, if satisfied with the suggested project name click on create fork.
+
+![](./docs/CreateFork.png)
+
+Next,
+
+![](./docs/CreateForkBtn.png)
+
+This would automatically create your workspace/repository.
+
+![](./docs/ForkedBranch.png)
+
+## Editor & IDE
 
-The browser also gives us things like localStorage, IndexedDB, Session Storage, Cookies and Cache Storage. Please make use of them for
+The repository contains folders with project appropriate settings and configurations for Visual Studio code, but beyond that, we are tool and editor/IDE agnostic so you can use whatever editor or IDE or you like.
 
-Please also make sure you use accessibility (A11y) with themes and styles paying attention to focus styling, contrast and keyboard-accesibility.
+Follow the instructions in [DEVSETUP](/docs/DEVSETUP.md) to setup your IDE workspace.
 
-### PR's
+## Prerequisites to use Database for programmers
 
-When you do a PR on GitHub, Please make sure you complete the section on the right.
+Adequate knowledge and familiarity with React 17 and or 18, JavaScript ES5/ES6/ES7.
+
+React gives us built-in hooks such as useEffect, useSelector, useRef, useHook, useCallback, useContext, useMemo, useForm, useHistory, etc. Ensure to make the most of these provided React built-ins.
+
+Also, use accessibility (A11y) with themes and styles paying attention to focus styling, contrast and keyboard-accesibility.
+
+Lastly, the browser gives us things like localStorage, IndexedDB, Session Storage, Cookies and Cache Storage.
+
+
+> Note: If assigned an issue and along the line you change your mind or can't figure it out, ensure to notify the team so it can be reassigned to another. <br><br>
```

Comment:
This section needs to be removed and included in the section about creating an issue.

Reply:
Okay i will do that.

Reply:
@Chizobaonorh Ok.